nntrainer/include/layer.h

```diff
@@ -194,18 +194,54 @@ class Layer {
    */
   virtual void copy(std::shared_ptr<Layer> l) = 0;
 
+  /**
+   * @brief     set Batch Normalization Layer followed
+   * @param[in] ok true/false
+   */
   void setBNfallow(bool ok) { this->bn_fallow = ok; }
```

Comment:
```suggestion
  void setBNfollow(bool ok) { this->bn_follow = ok; }
```
Is this typo? If it is, I might work on it.

Reply:
fixed. Thanks.